File: IntroPC/Tarea 6/Dectograd/dec2gsm.py
#! /usr/bin/env python  
import math
def dec2gsm(dec):
    
    grad= math.floor(dec)
    decimal=dec-grad
    g=grad
    a=decimal*60
    m=math.floor(a)
    b=a-m
    c=b*60
    s=c
    return g,m,s

File: IntroPC/Tarea 6/Dectograd/dec2gsm.cpp
#include <cmath>

void dec2gsm(double dec, double& g, double& m, double& s)
{
  double grad,decimal,a,b,c;
  grad= floor(dec);
  decimal=dec-grad;
  g=grad;
  a=decimal*60;
  m=floor(a);
  b=a-m;
  c=b*60;
  s=c;
}
